### DISCUSSION TITLE
Heatmap

### DISCUSSION THREAD

**Discussion by gsk7757** [biostars/biostar-handbook-issues#309] [Q&A]
When I use the rscript to generate a heatmap, it does generate a heatmap but picks genes from the entire list. May I know how these genes are picked and would it be possible to generate a heatmap for only differentially expressed genes that are up or down-regulated genes by at least 2-fold? Could you please help?

**Comment by ialbert** [comment]
it selects by FDR at 0.05, you can override that and apply other filters

**Comment by ialbert** [comment]
takes just a bit of R coding and you can even paste my code into the ChatGPT and ask it to give you a line that does what you want
